docs/glossary.md

```diff
@@ -87,6 +87,12 @@ Outputs from coinbase and stakebase transactions cannot be spent until the coinb
 
 Every block which is mined contains a single coinbase transaction. This transaction will only have one input, and that is newly created Decred which constitutes the PoW and Treasury portions of the block reward. Coinbase transactions are part of the regular transaction tree, which means that they will be rejected if proof-of-stake voters vote to reject the block which contains them. Decred created in coinbase transactions cannot be spent until the coinbase maturity period has passed.
 
+#### Coin type
+
+A constant, set for each cryptocoin for the BIP32 path. BIP-0032 introduces [HD Wallets](#hd-wallet). In the BIP32 path, every coin has an index. The index of Decred is 42. Registered coins are stored in a Satoshi Labs Improvement Proposal, in [SLIP-0044](https://github.com/satoshilabs/slips/blob/master/slip-0044.md). 
+
+Before SLIP44, the list of cryptocurrencies did not exist, the only 2 types were Bitcoin and Bitcoin Testnet. A wallet which contains addresses that were generated by the legacy BIP44 method, would contain addresses that the wallet software couldn’t see. 
```

Comment:
>method, would contain

Remove comma. While I agree a pause kind of makes sense here, it's not technically grammatically correct, and the sentence reads OK without the comma.